src/coreclr/jit/emitarm64.cpp

```diff
@@ -7987,6 +8035,14 @@ void emitter::emitIns_R_R_R(
             fmt = IF_DV_3A;
             break;
 
+        case INS_sve_uzp1:
+            // TODO: This could also be SVE_BR_3B or SVE_CI_3A
```

Comment:
We should standardize on `TODO-SVE:` for marking all SVE-related "to-do" work items.